src/allure-nunit-tests/allure-results/launch.json

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "Demo Allure tests"
```

Comment:
Didn't find where it goes with allure report UI - don't see the reason too keep this file